src/helpers/routes.js

```diff
@@ -145,7 +157,11 @@ export function makeRoutes (baseRoutes, {
       )
 
       if (shouldAddPrefix) {
-        path = `/${locale}${path}`
+        path = `/${locale.code}${path}`
```

Comment:
When running `no-lang-switcher`, `locale.code` is undefined, so I have to figure out a way of checking the type of locale (`object` for multiple-locales, and `string` for `no-lang-switcher`